src/components/Logo/BaseLogo.tsx

```diff
@@ -1,16 +1,16 @@
+import { BAD_SRCS } from 'constants/TokenLogoLookupTable'
 import { useState } from 'react'
 import { ImageProps } from 'rebass'
 import styled from 'styled-components/macro'
 
-const BAD_SRCS: { [tokenAddress: string]: true } = {}
-
 interface LogoProps extends Pick<ImageProps, 'style' | 'alt' | 'className'> {
   srcs: string[]
-  symbol?: string
+  src?: string | null
+  symbol?: string | null
   size?: string
 }
 
-const MissingImageLogo = styled.div<{ size?: string }>`
+export const MissingImageLogo = styled.div<{ size?: string }>`
```

Comment:
This is the only thing used from this file. Move it to AssetLogo, and rm BaseLogo.